### ISSUE TITLE
request new release

### ISSUE THREAD

**Issue by axel7083** [melonDS-emu/melonDS#1869]
The last release ([0.9.5](https://github.com/melonDS-emu/melonDS/releases/tag/0.9.5)) has been published on the Nov 3, 2022. Wouldn't be nice to have a more recent easy to access release ? Without the latest changes ? 

We can always download the artifact from the latest pipeline, but it is probably a bit tedious for non-developers ?

**Comment by sandwichwater** [comment]
next release isnt done yet

**Comment by Arisotura** [comment]
we're working on a next release, but it's going to be big

**Comment by Templayer** [comment]
https://github.com/melonDS-emu/melonDS/issues/1830#issuecomment-1721847050 this big? :D

**Comment by nadiaholmquist** [comment]
Nope, sorry. Next release will have additional black lines added to enhance the aesthetic.

**Comment by Templayer** [comment]
> Nope, sorry. Next release will have additional black lines added to enhance the aesthetic.
> 

![image](https://github.com/melonDS-emu/melonDS/assets/11728067/26f895dc-6c92-40bf-a090-5c5407ed02d1)


> For a more serious response, the next release will probably have the compute shader renderer which doesn't have that issue.

How much more processing power will that need? For what I am planning, I need to run two emulators at the same time while recording both. I can *just* handle that at smooth 60FPS right now, but if it is any harder on the processing, then that might not be a solution for me, unfortunately.

**Comment by Immersion95** [comment]
> Nope, sorry. Next release will have additional black lines added to enhance the aesthetic.
> 
> For a more serious response, the next release will probably have the compute shader renderer which doesn't have that issue.

I use the compute shader branch for weeks as my main NDS emulator and it's perfect. pmPerformance is flawless, input lag is finally back to real NDS level when upscaled and no graphical glitches with all my games :).

Fantastic work 👍

**Comment by Zer077** [comment]
> > Nope, sorry. Next release will have additional black lines added to enhance the aesthetic.
> > For a more serious response, the next release will probably have the compute shader renderer which doesn't have that issue.
> 
> I use the compute shader branch for weeks as my main NDS emulator and it's perfect. Performance is flawless, input lag is finally back to real NDS level when upscaled and no graphical glitches with all my games :).
> 
> Fantastic work 👍

Yeah, I play on my Steam Deck with compute shader renderer and is perfect!

**Comment by Templayer** [comment]
> > > Nope, sorry. Next release will have additional black lines added to enhance the aesthetic.
> > > For a more serious response, the next release will probably have the compute shader renderer which doesn't have that issue.
> > 
> > 
> > I use the compute shader branch for weeks as my main NDS emulator and it's perfect. Performance is flawless, input lag is finally back to real NDS level when upscaled and no graphical glitches with all my games :).
> > Fantastic work 👍
> 
> Yeah, I play on my Steam Deck with compute shader renderer and is perfect!

Can't wait for it to fuck it up for me in a spectacular way. :D  

It sounds too good to be true.

But hey, if it does work as some people seemingly say, then it is awesome.

**Comment by sandwichwater** [comment]
> Can't wait for it to fuck it up for me in a spectacular way. :D
> 
> It sounds too good to be true.
> 
> But hey, if it does work as some people seemingly say, then it is awesome.

maybe not the news you wanna hear but atleast for me i experience a lot of texture alignment issues compared to the opengl renderer , thought i am on windows rather on steam deck so that may be why🤔 either way we will have to see when the official release happens

**Comment by Templayer** [comment]
> > Can't wait for it to fuck it up for me in a spectacular way. :D
> > It sounds too good to be true.
> > But hey, if it does work as some people seemingly say, then it is awesome.
> 
> maybe not the news you wanna hear but atleast for me i experience a lot of texture alignment issues compared to the opengl renderer , thought i am on windows rather on steam deck so that may be why🤔 either way we will have to see when the official release happens

Ruins my hopes for a seamless Pokémon Heart Gold a bit. We will see.

**Comment by nadiaholmquist** [comment]
Could you please elaborate on those texture issues? Maybe post some screenshots in a separate issue?

Some level of janky rendering is normal and expected for DS games, it happens on hardware too. Maybe it could be that is what you're seeing?

**Comment by TathagataDM** [comment]
[This one](https://github.com/melonDS-emu/melonDS/issues/1306) is big for me, though it occurs in both GL and the compute renderer.
